### ISSUE TITLE
shapeerror

### ISSUE THREAD

**Issue by wwood** [rhysnewell/rosella#47]
Running this from aviary (which didn't error out as probably should have)

```
[2023-12-01T19:27:11Z INFO  rosella] rosella version 0.5.1
[2023-12-01T19:27:11Z INFO  rosella::recover::recover_engine] Calculating contig coverages.
[2023-12-01T19:27:16Z INFO  rosella::recover::recover_engine] Calculating TNF table.
[2023-12-01T19:29:39Z ERROR rosella] Recover Failed with error: ShapeError/IncompatibleShape: incompatible shapes
```

What more info did you want?

**Comment by rhysnewell** [comment]
I think I have a solution to this if either of you is willing to share your assembly with me I can test whether it is fixed and push out a new release. Otherwise, potential fix is on branch #ISS-47

**Comment by sjparkmk** [comment]
> I think I have a solution to this if either of you is willing to share your assembly with me I can test whether it is fixed and push out a new release. Otherwise, potential fix is on branch #ISS-47

thanks for your reply.. but how could i share the huge file (ca. 7.6 Gb) to you?

**Comment by rhysnewell** [comment]
Very true, that won't work. Are you willing to try and clone the branch I mentioned and install via  cargo? Install instructions should outline how, otherwise I will continue trying to reproduce the error on my end

**Comment by rhysnewell** [comment]
I was able to reproduce the issue and have fixed in v0.5.2 
This new version should be available via conda soon
